app/assets/stylesheets/doctor_sessions.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the doctor_sessions controller here.
```

Comment:
if you don't use these xxx.scss files please delete it. you can always create it when you need it